frontend/store/token.js

```diff
@@ -143,7 +165,22 @@ export const actions = {
     const configLengths = parseInt(await atd.methods.distributionConfigsLength().call())
     const distributionConfigs = await Promise.all(new Array(configLengths).fill().map((_, i) => atd.methods.distributionConfigs(i).call()))
     distributionConfigs.forEach(distributionConfig => commit('addDistributionConfigs', { distributionConfig }))
-    console.log(distributionConfigs)
     return distributionConfigs
+  },
+  FETCH_ALLOWANCES: async ({ dispatch, commit }) => {
+    commit('resetAllowances')
+    const distributionConfigs = await dispatch('FETCH_DISTRIBUTION_CONFIGS')
+    const address = atd_contracts.find(contract => contract.to === 'latest').address
+    const token = (await artifacts).token
+    const allowancesPromise = distributionConfigs.map(distributionConfig => ({ ...distributionConfig, value: token.methods.allowance(distributionConfig.lockedWallet, address).call() }))
+    const allowances = await Promise.all(
+      allowancesPromise.map(allowance =>
+        allowance.value.then(value => {
+          allowance.value = new BigNumber(value).div(PRECISION).toNumber()
+          return allowance
+        })
+      )
+    ).forEach(allowance => commit('addAllowance', { allowance }))
```

Comment:
I don't think forEach return anything. I think `allowances` is null here.
```suggestion
    )
    allowances.forEach(allowance => commit('addAllowance', { allowance }))
```